contracts/utils/Base64.sol

```diff
@@ -87,4 +89,68 @@ library Base64 {
 
         return result;
     }
+
+    /**
+     * @dev Converts a `bytes` to its Bytes64Url `string` representation.
+     */
+    function encodeUrl(bytes memory data) internal pure returns (string memory) {
```

Comment:
I think this would be better. Also consistent with the naming of other functions like `baseURI` in ERC721./

```suggestion
    function encodeURL(bytes memory data) internal pure returns (string memory) {
```